winterfell/models/form.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+var _ = require('lodash');
+var FormProps = require('../utils/formProperties');
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+var FormSchema = new Schema({
+  _id: String,
```

Comment:
_id is a special field in mongodb, with type ObjectID (not String), I'm not sure how mongoose allows this is must be renaming or something on the backend, but we should not make models that override _id, its confusing